incl/macros.html

```diff
@@ -1,12 +1,14 @@
 <#def RELEASES>https://github.com/coq/coq/releases</#def>
 <#def BASEURL>https://github.com/coq/coq/releases/tag/</#def>
 
-<#def CURRENTMAJORVERSION>8.9</#def>
-<#def CURRENTVERSION>8.9.1</#def>
-<#def CURRENTVERSIONDATE>May 2019</#def>
-<#def CURRENTVERSIONTAG>V8.9.1</#def>
+<#def CURRENTMAJORVERSION>8.10</#def>
+<#def CURRENTVERSION>8.10.0</#def>
+<#def CURRENTVERSIONDATE>October 2019</#def>
+<#def CURRENTVERSIONTAG>V8.10.0</#def>
 <#def CURRENTVERSIONURL><#BASEURL><#CURRENTVERSIONTAG></#def>
 <#def CURRENTCREDITSURL>https://github.com/coq/coq/blob/<#CURRENTVERSIONTAG>/CREDITS</#def>
 
-<#def UPCOMINGVERSION>8.10</#def>
-<#def UPCOMINGVERSIONURL>https://github.com/coq/coq/releases/tag/V8.10%2Bbeta3</#def>
+<#ifdef UPCOMINGVERSION>
+<#def UPCOMINGVERSION>8.11</#def>
+<#def UPCOMINGVERSIONURL>https://github.com/coq/coq/releases/tag/V8.11%2Bbeta1</#def>
```

Comment:
Without any change to the main page, this is going to introduce broken and inaccurate information.

Reply:
What change do you want in the main page? “Don’t read the source code of this website as it may contain inaccurate data.”

Reply:
Moreover I fail to see anything inaccurate there.

Reply:
Sorry for the noise. I should have compiled the website before making any comment. I had forgotten about the recent enhancements by Guillaume that made this possible.